### PR TITLE
Omniverse Docs: Adding support for CE 2022.1

### PR DESCRIPTION
**Pull request by mistafunk**
Also adding note regading live mode changes.

### CODE REVIEW

File: documentation/omniverse/index.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ The Omniverse Connector for CityEngine provides an uni-directional link to Omniv
 
 ## Installation Instructions
 
-1. Install CityEngine 2021.1 or 2022.0. Please refer to [CityEngine - Get Started](https://doc.arcgis.com/en/cityengine/2022.0/get-started/cityengine-get-started.htm) for details.
-1. Install the connector from the corresponding Omniverse Launcher item, see [Omniverse Launcher Overview](https://docs.omniverse.nvidia.com/prod_launcher/prod_launcher/overview.html) for additional information.
+1. Ensure one (or more) of CityEngine 2021.1, 2022.0 or 2022.1 is installed. Please refer to [CityEngine - Get Started](https://doc.arcgis.com/en/cityengine/2022.0/get-started/cityengine-get-started.htm) for installation instructions.
```
Comment (chr11115) [inline documentation/omniverse/index.md:19]: The first sentence sounds a bit off, how about something along the lines of
"Install at least one of the following versions of CityEngine on your system: 2021.1, 2022.0, or 2022.1."?

**Review by chr11115 — APPROVED** [review]
Almost good, just a minor language nag

File: documentation/omniverse/index.md (hunk 1 above)
Comment (mistafunk) [reply documentation/omniverse/index.md:19]: good point, adjusted

**Review by chr11115 — APPROVED** [review]

**Comment by mistafunk** [comment]
This branch will be merged once Nvidia confirms publication of the connector update in the Omniverse Launcher

**Pull request merged**